MdeModulePkg/Core/Dxe/Misc/MemoryProtectionSupport.c

```diff
@@ -25,10 +25,10 @@ MEMORY_PROTECTION_SPECIAL_REGION_PRIVATE_LIST_HEAD  mSpecialMemoryRegionsPrivate
   INITIALIZE_LIST_HEAD_VARIABLE (mSpecialMemoryRegionsPrivate.SpecialRegionList)
 };
 
-BOOLEAN                        mIsSystemNxCompatible       = TRUE;
-EFI_MEMORY_ATTRIBUTE_PROTOCOL  *MemoryAttributeProtocol    = NULL;
-UINT8                          *mBitmapGlobal              = NULL;
-LIST_ENTRY                     **mArrayOfListEntryPointers = NULL;
+BOOLEAN                        mEnhancedMemoryProtectionActive = TRUE;
+EFI_MEMORY_ATTRIBUTE_PROTOCOL  *MemoryAttributeProtocol        = NULL;

```

Comment:
Not essential here, but at some point it would be nice to name this `mMemoryAttributeProtocol` per convention and for consistency.

Reply:
I added another commit to this series to rename the global